### PR TITLE
Badges

### PR DESCRIPTION
**Pull request by ckahub**
badge1
Added badges with status 
added badeges with number

### CODE REVIEW

File: Components/Badges/badges.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
+@import url('../../styles/utility.css');
+@import url('../../styles/navBar.css');
+@import url('../../styles/sideBar.css');
+@import url('../../styles/variable.css');
+@import url('../Avatar/avatar.css');
+
```
Comment (anandkumarcoder) [inline Components/Badges/badges.css:6]: suggestion You make a separate file where you keep all the imported external sheets (main.css) then use this main.css in doc.css.

File: styles/sideBar.css [hunk 2]
```diff
@@ -4,13 +4,37 @@
 
 .sidebar{
     font-family: 'Supermercado One', cursive;
-    flex: 0 0 240px;
+    flex: 0 0 20%;
     margin-right: 1rem;
-    background-color: #fff;
+    /*background-color:skyblue;
+    /*width: fit-content;
+    height:max-content;  */
```
Comment (anandkumarcoder) [inline styles/sideBar.css:11]: keep the code clean. delete commented out codes

File: styles/variable.css [hunk 3]
```diff
@@ -10,6 +15,7 @@
     --warning-color : #FF9800;
     --active-color : #1EA7FD;
     --off-white : #F6F9FC;
+    --dnd-color :grey;
```
Comment (anandkumarcoder) [inline styles/variable.css:18]: use hex code

File: styles/utility.css [hunk 4]
```diff
@@ -21,6 +21,7 @@ body{
     margin: 0px auto;
     max-width: 800px;
```
Comment (anandkumarcoder) [inline styles/utility.css:22]: you have used rem in other places. try to use rem it will help in responsiveness

**Review by anandkumarcoder — COMMENTED** [review]
try to make changes in asked in the comment

File: Components/Badges/badges.css (hunk 1 above)
Comment (ckahub) [reply Components/Badges/badges.css:6]: thankyou for suggestion

File: styles/sideBar.css (hunk 2 above)
Comment (ckahub) [reply styles/sideBar.css:11]: ok i will do so

File: styles/variable.css (hunk 3 above)
Comment (ckahub) [reply styles/variable.css:18]: ok i will do so

**Pull request merged**